hardening/image-builder/test.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+
+import os
+
+from lib import results, versions, oscap, osbuild
+
+
+osbuild.Host.setup()
+
+g = osbuild.Guest()
+
+profile = os.environ['PROFILE']
+
+g.create(profile=profile)
+
+profile = f'xccdf_org.ssgproject.content_profile_{profile}'
+
+with g.booted():
+    # old RHEL-7 oscap mixes errors into --progress rule names without a newline
+    verbose = '--verbose INFO' if versions.oscap >= 1.3 else ''
+    redir = '2>&1' if versions.oscap >= 1.3 else ''
+    # RHEL-7 HTML report doesn't contain OVAL findings by default
+    oval_results = '' if versions.oscap >= 1.3 else '--results results.xml --oval-results'
```

Comment:
We can skip the RHEL7 workarounds as we know there will be no IB for RHEL7.

Reply:
See my comment about `main.fmf`, I applied the same logic here.